_infra/helm/locust/locustfiles/requests.json

```diff
@@ -7,13 +7,49 @@
         },
         {
             "method": "GET",
-            "url": "/surveys/history",
-            "expected_response_text": "<title>Completed surveys | ONS Business Surveys</title>"
+            "harvest_url": {
+                "id": "create-message-link-1",
+                "link_text": "Get help with this survey"
+            },
+            "expected_response_text": "Choose an option",
+            "grouping": "/help"
```

Comment:
I have grouped the help path (POSTs) as all the same, they are simple checkbox new page based on decision pattern and as such don't need to be separate

Reply:
Agreed.. these are fairly superficial (related) POST requests.